lib/api.js

```diff
@@ -46,7 +46,7 @@ function getGist(uid) {
   return gistClient
     .get(`/gists/${uid}`)
     .then(res => res.data)
-    .then(({ owner, files }) => {
+    .then(({ files }) => {
```

Comment:
@raboid i still want to be able to access the gist_id and owner in the future though, similar to: https://github.com/dawnlabs/carbon/pull/713/files#diff-34b1810563417208c53b912bd37bea3bR19